### PR TITLE
Display sandbox layout to sandbox users

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

- Adds a hook to return the user's account type. 
- Displays the `SandboxLayout` to sandbox users who have completed onboarding.
- Adds a fallback to `getInitials` function to resolve an error experienced during development.
- Removes duplicate `getInitials` function.
- Resolves error where the `org name` didn't appear in the `Sidebar` after completing onboarding by invalidating the `org list` and `org detail` queries.

Note: A test will be added in a separate PR.

Fixes SC-23123, SC-23125, & SC-23347

### Type of change

- [x] new feature
- [x] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/24094232?key=a812e61e398631659e307c58a6e24836

### Definition of Done

- [ ] I have manually tested the change running it locally (having rebuilt all containers) or via unit tests 
- [ ] I have added unit and/or integration tests that cover my changes
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have updated the dependencies list if necessary (including updating yarn.lock and/or go.sum)
- [ ] I have recompiled and included new protocol buffers to reflect changes I made if necessary
- [ ] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ] I have notified the reviewer via Shortcut or Slack that this is ready for review
- [ ] Front-end: Checked sm, md, lg screen resolutions for effective responsiveness
- [ ] Backend-end: Documented service configuration changes or created related devops stories

### Reviewer(s) checklist

- [ ] Front-end: I've reviewed the Figma design and confirmed that changes match the spec.
- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/utils/getInitials.ts [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-export default function getInitials(name = '') {
-  return name
-    .match(/(^\S\S?|\b\S)?/g)
-    ?.join('')
-    .match(/(^\S|\S$)?/g)
-    ?.join('')
-    .toUpperCase();
-}
+export const getInitials = (name: string) => {
+  const nameArray = name?.split(' ') || [''];
```
Comment (daniellemaxwell) [inline web/beacon-app/src/utils/getInitials.ts:2]: While toggling between onboarded and not onboarded user states, I kept getting an undefined error related to this function. Using optional chaining  `(? after name)`  didn't completely resolve it, so `['']` was added as a fallback.

File: web/beacon-app/src/features/members/hooks/useUpdateProfile.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ export function useUpdateProfile(): ProfileUpdateMutation {
       queryClient.invalidateQueries({ queryKey: [RQK.MEMBER_LIST] });
       queryClient.invalidateQueries({ queryKey: [RQK.MEMBER_DETAIL] });
       queryClient.invalidateQueries({ queryKey: [RQK.PROFILE] });
+      queryClient.invalidateQueries({ queryKey: [RQK.ORG_DETAIL] });
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/members/hooks/useUpdateProfile.tsx:19]: This change informs React Query that the data for these queries is now stale, due to a user completing the onboarding process, and should be re-fetched.

File: web/beacon-app/src/utils/getInitials.ts (hunk 1 above)
Comment (pdamodaran) [reply web/beacon-app/src/utils/getInitials.ts:2]: the updated function is easier to follow as well - thanks for making this fix!

File: web/beacon-app/src/features/members/hooks/useUpdateProfile.tsx (hunk 2 above)
Comment (pdamodaran) [reply web/beacon-app/src/features/members/hooks/useUpdateProfile.tsx:19]: Thanks for noticing the issue and making the fix!

**Review by pdamodaran — APPROVED** [review]
looks good @daniellemaxwell!

**Pull request merged**